homeassistant/components/vicare/sensor.py

```diff
@@ -189,10 +189,74 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         state_class=SensorStateClass.MEASUREMENT,
     ),
     ViCareSensorEntityDescription(
-        key="solar power production",
-        name="Solar Power Production",
+        key="solar power production today",
```

Comment:
This changed the description key which is used in the sensor unique_id. We can't change the unique_id without a migration of existing entity entries in the entity registry. Do we need to change the key?

Reply:
This is not changing the key. It's just deleting the key "solar power production", which was not providing the data, the user expected. Instead of that, this adds a new key "solar power production today", which now provides the value the users expects.

Reply:
Ok. Removing a sensor is a breaking change. Please add a breaking change paragraph in the PR description. Briefly describe what changed and how the users should cope with the change.

Reply:
I changed the desciption. Could you please have a look at this again.
THX